docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3.8"
+version: "3.7"
```

Comment:
🤔 why the downgrade?

Reply:
Because it didn't work for me locally and I was lazy:

```
ERROR: Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
```

I'll try to fix this.

Reply:
I have fixed this now! Turns out `docker compose != docker-compose`.